dj/models/node.py

```diff
@@ -461,7 +465,7 @@ def extra_validation(self) -> None:
         """
         Extra validation for node data.
         """
-        if self.type in (NodeType.SOURCE, NodeType.CUBE):
+        if self.type in (NodeType.SOURCE,):
```

Comment:
I'm curious why cubes are removed from this check. They shouldn't have a query attribute right?

Reply:
So in this case I've added the generated query for a cube to `query`, which triggers this extra_validation check to fail. But it shouldn't be in the user input, so this check might need to be on the creation input model.

Reply:
I see, makes sense!